data/models.py

```diff
@@ -285,6 +285,10 @@ class Student(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name="students")
     key = URLKeyField(max_length=64, help_text="Alphanumeric student id")
 
+    name = models.CharField(max_length=64, blank=True, default='No Name', help_text="Full name of the student")
+    email = models.EmailField(blank=True, default='No Email')
```

Comment:
Don't forget to add the Django database migrations when you are finalizing the pull request.

`python manage.py makemigrations`

The migration Python files must be committed to git.